### DISCUSSION TITLE
Field validation not working on Text Field

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#4843] [Help] [answered]
Hello,

![image](https://user-images.githubusercontent.com/91942877/157260916-b1c50f4c-818b-4dc5-aa02-d6cf7a5f3efb.png)
When using a text field, the schema validation rules doesn't work properly. The form validates even with an empty value.
If I add manually the same rule (see screenshot), the field is required as expected.
The schema validation rules is working for Long Form Fields without custom rules in the same form.
Budibase version : 1.0.73

Did I miss something?

Thank you!

**Comment by mjashanks** [comment, marked as answer]
Hey @diogenesbrussels - I am not experiencing this. Have tried this with Postgres and Budiabse DB. I'll give it a go on MySQL as well.

Have tried for Text, Long Text, Number - all seem to work ok for me.

I also am using `v1.0.73`

I did find a bug ( I think) whereby you can delete a schema-required field from a form and the record is saved, without error.

**Reply by mjashanks** [reply]
Also works as expected in MySQL.

However, I did initially get tripped up as I did not refresh my app after making the column required and updating the form field. Not sure if that's the case with you - just throwing it out there!

**Reply by diogenesbrussels** [reply]
I did test with a refresh but it isn't working either.
I made a small video to show what is happening:
https://user-images.githubusercontent.com/91942877/158346725-29100db3-5346-4fc9-8115-a3c5f2b6fd83.mp4 

Do not hesitate to ask me for more information if necessary.

**Reply by mjashanks** [reply]
Ah - thank you! This is a bug.

It only seems to be an issue with the table "Display Column". I'll get this logged.
